.github/workflows/python-code-quality.yml

```diff
@@ -114,7 +114,10 @@ jobs:
       - name: Run Pylint on other files using pytest
         run: |
           pip install pytest pytest-pylint==0.19
-          echo "::warning file=.github/workflows/python-code-quality.yml,line=116,col=42,endColumn=48::Temporarily downgraded pytest-pylint to allow merging other PRs. The errors reported with a newer version seem legitimite and should be fixed (2023-10-18, see https://github.com/OSGeo/grass/pull/3205)"
+          echo "::warning file=.github/workflows/python-code-quality.yml,line=116,col=42,endColumn=48::\
+            Temporarily downgraded pytest-pylint to allow merging other PRs. The errors reported\
+            with a newer version seem legitimite and should be fixed (2023-10-18,\
+            see https://github.com/OSGeo/grass/pull/3205)"
```

Comment:
It breaks the output of the warning, but that warning was written by me to intentionally mentioning that the test failures must be fixed one day to upgrade the package version.

Reply:
It can stay as is!